app/src/test/java/com/example/benchproject/data/popular/movies/entity/MovieResponseRemoteTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.benchproject.data.popular.movies.entity
+
+import com.example.benchproject.domain.popular.movies.entity.MovieModel
+import org.amshove.kluent.shouldBeEqualTo
+import org.junit.jupiter.api.Test
+
+internal class MovieResponseRemoteTest {
+    private val moviesRemote = listOf(
```

Comment:
One general rule, there is no need for a private modifier in tests :)
https://medium.com/android-news/clean-tests-part-3-more-about-names-ce9104093941

Reply:
Ooo ok :D I understand and I delete `private` in any file where I could in this PR

Reply:
And I will do the same for all tests in possible PR or in new PR for this :D